data/strings/strings.txt

```diff
@@ -24030,7 +24030,7 @@
     ca = Definiu les vies a evitar
     cs = Definovat silnice, kterým se vyhnout
     da = Definer veje du vil undgå
-    de = Umwege einstellen
+    de = Definieren Sie zu vermeidende Straßen
```

Comment:
"Routenbeschränkung einstellen"

This switched back to "Umwege" several times for some reason...

Reply:
OK. Changed to "Routenbeschränkung einstellen". Please let me know if you'd prefer Umwege. Maybe is shorter but it's not a precise translation.